config.toml

```diff
@@ -36,9 +36,10 @@ sourceRepo = "https://github.com/o3de/o3de"
 # Docs version data format: ["display name", "baseURL"]
 # Note: A version that has "www.o3de.org" in its baseURL will get " (latest)" appended to its display name in the version selector.
 versions = [
-  ["22.10", "https://www.o3de.org"],
+  ["23.05", "https://www.o3de.org"],
   ["development", "https://development--o3deorg.netlify.app"],
-  ["22.05", "https://2205--o3deorg.netlify.app"]
+  ["22.10", "https://version-2210--o3deorg.netlify.app"],
+  ["22.05", "https://version-2205--o3deorg.netlify.app"]
```

Comment:
Note that this updated URL for the archived version 22.05 does not work yet. I think that's because no changes have been published to that branch since it was moved from `2205` to `version/2205`. Upon release of 23.05, we should make sure to cherry pick the commit for this PR into that newer branch so that the deploy URL corrects itself.